File: Atividade 1/1.4/tarefa.py
def fatorial(number):
	fatorial = 1
	for i in range(2,number+1):
		fatorial = fatorial * i
	return fatorial

number = int(input("Digite um numero: "))
print("Numero digitado: {}.".format(number))
print("Fatorial deste numero: {}.".format(fatorial(number)))

File: Atividade 1/1.4/Tarefa.java
import java.util.Scanner;

public class Tarefa {
	public static void main(String[] args) {
		int number;
		Scanner in = new Scanner(System.in);
		
		System.out.printf("Digite um numero: ");
		number = in.nextInt();
		int fatorial = fatorial(number);
	
		System.out.printf("Numero digitado: " + number + ".\n");
		System.out.printf("Fatorial deste numero: " + fatorial + ".\n");
	}

	public static int fatorial(final int number) {
		int fatorial = 1;
		for(int i=2; i<=number; i++){
			fatorial *= i;
		}
		return fatorial;
	}
}
